ecosystem/sep-0012.md

```diff
@@ -54,22 +54,22 @@ Alternatively, if the client cannot add the authorization header. The JWT should
 
 Checks the status of a customers information. This allows clients to check whether the customers information was accepted, rejected, or still needs more info.  Lookups are done via the stellar account ID and memo that were used for the initial PUT request.
 
-This also provides the information needed from a KYC provider to KYC a user.  If a new account that has never been KYC'd is fetched,the server should respond with `NEEDS_INFO` and a list of all the fields the client will need to provide.
+This also provides the information needed from a KYC provider to KYC a customer.  If a new account that has never been KYC'd is fetched,the server should respond with `NEEDS_INFO` and a list of all the fields the client will need to provide.
 
 ### Request
```

Comment:
More of a general SEP-12 question, since all the request fields are optional what is the expected response if the GET was just `GET [KYC_SERVER]/customer`?

Reply:
Thats a good question because it points out that the SEP isn't clear.

To answer your question: `GET /customer` would return the fields required for the default `type` of customer. If no identifying parameters are sent by the client, the endpoint should assume its a request for a new customer that doesn't yet exist.

There are 2 purposes to the `GET /customer` endpoint:
- Fetch the status of an existing customer
- Fetch the fields required for a new customer

If the sending anchor is registering a customer that is _new to the receiving anchor_, then there's no reason to pass parameters, because the receiving anchor doesn't have the customer record in their DB yet.

If the sending anchor is checking on the status of an existing customer, the (`id`) or (`memo`, `memo_type`) parameters should be used. Both can identify the customer. But if the sender doesn't have an `id` returned previously from the receiving anchor, it can only use `memo` and `memo_type`.

Reply:
I'll add more to the SEP to clarify that.